spirv_reflect.h

```diff
@@ -1450,6 +1663,7 @@ class ShaderModule {
   SpvReflectResult GetResult() const;
 
   const SpvReflectShaderModule& GetShaderModule() const;
+  Evaluation GetEvaluation() const;
```

Comment:
Constant module can retrieve non-const evaluation interface. Evaluation state should be mutable even after module is fully parsed. (on-demand evaluation, set constant, re-evaluate)